app/controllers/general.php

```diff
@@ -612,6 +612,7 @@
     });
 
 include_once __DIR__ . '/shared/api.php';
+include_once __DIR__ . '/shared/cache.php';
```

Comment:
Seems like we're not using this controller anymore. We can also remove all the `cache` groups from the different routes.

Reply:
removed